package.json

```diff
@@ -26,40 +26,44 @@
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/gxa/my-package.git"
+    "url": "https://github.com/ebi-gene-expression-group/scxa-home-landing-page"
   },
   "dependencies": {
-    "atlas-homepage-cards": "file:../atlas-homepage-cards",
-    "file-loader": "^2.0.0",
+    "atlas-homepage-cards": "^1.2.0",
+    "atlas-latest-experiments-cards": "^1.2.1",
     "prop-types": "^15.6.2",
     "react": "^16.4.1",
     "react-dom": "^16.4.1",
     "react-router": "^4.3.1",
     "react-router-dom": "^4.3.1",
-    "urijs": "^1.19.1",
-    "url-loader": "^1.1.2"
+    "styled-components": "^4.1.2",
+    "urijs": "^1.19.1"
   },
   "devDependencies": {
-    "ajv": "^6.5.2",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
-    "babel-loader": "^7.1.5",
-    "babel-plugin-transform-object-rest-spread": "^6.26.0",
-    "babel-polyfill": "^6.26.0",
-    "babel-preset-env": "^1.7.0",
-    "babel-preset-react": "^6.24.1",
+    "@babel/cli": "^7.1.2",
+    "@babel/core": "^7.1.2",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.0.0",
+    "ajv": "^6.5.4",
```

Comment:
This one was included because of a dependency warning with old Babel (6.x). It can be removed now.